File: minseon/22_11_24_01.py
#외계행성의 나이
def solution(age):
    answer = ''
    a = str(age)
    age_list = {0:'a' , 1:'b', 2:'c', 3:'d', 4:'e', 5:'f', 6:'g', 7:'h', 8:'i', 9:'j' }
    for i in range(len(a)):
        answer += age_list.get(int(a[i]))
    return answer

File: seungbeom/22_11_24_01.java
// https://school.programmers.co.kr/learn/courses/30/lessons/120834

class Solution {
    public String solution(int age) {
        StringBuilder ageConverter = new StringBuilder();
        while (age != 0) {
            ageConverter.insert(0, (char) (age % 10 + 97));
            age = age / 10;
        }
        return ageConverter.toString();
    }
}